packages/plugin-ext/src/plugin/languages/lens.ts

```diff
@@ -1,5 +1,5 @@
 /********************************************************************************
- * Copyright (C) 2018 Red Hat, Inc. and others.
```

Comment:
always preserve an year of the original contribution, if you want to show current year, use `2018-2019`, please make sure that PR does not have such changes in other places

Reply:
OK. I'll fix